README.md

```diff
@@ -7,11 +7,9 @@ Ensure you have Python 2 or 3 and pip (`easy_install pip`) and then:
 
 ```shell
 pip install mintapi
-brew cask install chromedriver # or sudo apt-get install chromium-chromedriver on Ubuntu/Debian
 ```
 
-Note that chromedriver must be version 59+ if you want to use headless mode. If not installing via pip,
-make sure to install the `install_requires` dependencies from setup.py yourself.
+`mintapi` does it scraping by driving an instance of Chrome (or chromium) and navigating, as a human user would, thought the login flows. Once logged in, the API allows programatic access to various Mint REST APIs. Selenium/WebDriver is used to accomplish this, and specifically, ChromeDriver under the hood. `mintapi` will download the latest stable release of chromedriver, unless --use_chromedriver_on_path is given. **NOTE: You must have [Chrome](https://www.google.com/chrome/) or [Chromium](https://www.chromium.org/getting-involved/dev-channel/) installed, and on the `stable` track.**
```

Comment:
`thought the login flows`. I assume this is supposed to be throughout, but I think this sentence fragment can just be removed. It is clear as is :)

Is the requirement to have Chrome or Chromium installed yourself new or just being noted more clearly now?

Reply:
Revised that sentence for clarity. Good catch!

The requirement to have chrome/chromium installed (and in a standard location) is not new. The *new* requirement introduced by this change, and very intentionally, is that you must have an up-to-date version of chrome. This is so the chromedriver version can match the chrome version. This is for the best, as we don't want users staying behind on old versions of chrome when connecting to sensitive sites. For advanced users that _do_ want this capability, they can use the new path argument param to stay behind/ahead of stable. 

You could imagine someone adding additional support for the [remaining supported selenium drivers](https://selenium-python.readthedocs.io/getting-started.html#example-explained) (Firefox, IE/Edge, [Safari has their own](https://developer.apple.com/documentation/webkit/testing_with_webdriver_in_safari)). That, coupled with some crazy platform specific checks to detect which browsers are installed, and what the system default browser is, could be used to auto-detect the right driver to use. But that's a large burden on this module to support (seems like a good idea for a new python project, maybe?), especially since the DOM pieces could need special flows on a per browser X platform basis.